app/js/searchFunction.js

```diff
@@ -26,9 +29,37 @@ function sanitizeInput (input) {
     return input.trim()
 }
 
+/**
+ *
+ * returns the results from the search input as an array, allows partial search
+ *
+ * @param array from AJAX request
+ *
+ * @returns array of objects
+ */
+
+function search (copy) {
```

Comment:
With a semantic name, it will be better. This is not the copy of the data anymore, we might call it data or resultAjaxRequest